### PR TITLE
Add ability to send transaction ping batches

### PR DESCRIPTION
**Pull request by carllin**
#### Problem
Current ping messages have to be tried one at a time, and don't support batch pings where we send `N` pings at once which tests a different workload

#### Summary of Changes
Introduce a `num_batches` and `batch_size` options to allow configurable batches

Fixes #

### CODE REVIEW

File: cli/src/cluster_query.rs [hunk 1]
```diff
@@ -1401,32 +1413,45 @@ pub fn process_ping(
...
             false,
             SpendAmount::Some(lamports),
             &blockhash,
             &config.signers[0].pubkey(),
             build_message,
             config.commitment,
-        )?;
-        let mut tx = Transaction::new_unsigned(message);
-        tx.try_sign(&config.signers, blockhash)?;
+        )?;*/
```
Comment (carllin) [inline cli/src/cluster_query.rs:1444]: @pieceofr if you could take over this PR and properly resolve this, that would be great, thanks!

File: cli/src/cluster_query.rs [hunk 2]
```diff
@@ -248,12 +248,12 @@ impl ClusterQuerySubCommands for App<'_, '_> {
                         .help("Wait interval seconds between submitting the next transaction"),
                 )
                 .arg(
-                    Arg::with_name("count")
-                        .short("c")
-                        .long("count")
```
Comment (CriesofCarrots) [inline cli/src/cluster_query.rs:253]: Please let's not break users like this. Probably `count` could be an alias of `num_batches`, if you don't want to handle the parameters separately.

Comment (carllin) [reply cli/src/cluster_query.rs:253]: don't worry, I don't think I'm checking this in :)

Comment (taozhu-chicago) [reply cli/src/cluster_query.rs:253]: I cherry-picked this change into local branch, ready to run 'batch' test if you desire @carllin

**Comment by carllin** [comment]
@taozhu-chicago yeah, let's run it, no guarantee it's correct though 😛

**Pull request closed without merging**